ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/BaseLoadBalancer.java

```diff
@@ -578,6 +578,16 @@ public Server getServerByIndex(int index, boolean availableOnly) {
     }
 
     @Override
+    public List<Server> getReachableServers() {

```

Comment:
Let's mark getServerList(boolean) as deprecated.  Move the real code into getAllServers() and getReachableServers() and call them from getServerList()